delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/resolvers/ResolverResolutionRejection.scala

```diff
@@ -19,15 +19,42 @@ sealed abstract class ResolverResolutionRejection(val reason: String) extends Pr
 
 object ResolverResolutionRejection {
 
+  /**
+    * Rejection when the access to the given project has been denied by using the identity resolution
+    * defined in the resolver
+    * @param projectRef          the project where we attempted the resolution
+    * @param identityResolution  the identity resolution used
+    */
+  final case class ProjectAccessDenied(projectRef: ProjectRef, identityResolution: IdentityResolution)
+      extends ResolverResolutionRejection(
+        s"Access to project '$projectRef' has been denied using resolution $identityResolution"
```

Comment:
The message here should be as similar as possible as the one in `AuthorizationFailed` since it is the same for the client.